### DISCUSSION TITLE
Is there a concrete example of using a Service Account to access the API?

### DISCUSSION THREAD

**Discussion by trevor-vaughan** [NUWCDIVNPT/stig-manager#964] [Q&A]
[Section 2.1.2 of the docs](http://localhost:54000/docs/installation-and-setup/authentication.html#service-account-client-setup) indicates that you can use a keycloak service account for accessing the STIGMan API.

I'm hoping that someone can point me to a functional example of this for reference.

**Comment by cd-rite** [comment]
Hi @trevor-vaughan  Our "Watcher" client uses a service account to update collections: https://github.com/NUWCDIVNPT/stigman-watcher
This will require some configuration in Keycloak or your OIDC provider.
The wiki in that repo covers some of this topic (https://github.com/NUWCDIVNPT/stigman-watcher/wiki/Requirements-and-Configuration), and we have a video here explaining the Keycloak config: https://youtu.be/blv_nuONnTw

**Reply by trevor-vaughan** [reply]
@cd-rite Thanks! I really appreciate the fast response.
